bgcompute.py

```diff
@@ -57,6 +58,7 @@ def bgcompute_update_election_jurisdictions_file() -> int:
 
 
 def bgcompute_update_ballot_manifest_file() -> int:
+    # pylint: disable=singleton-comparison
```

Comment:
You might be wondering why there are some places where I disable this rule. It's because sqlalchemy filters do not work with `… is None`, but require `== None`. Alternatively, [we could do `.is_(None)`](https://docs.sqlalchemy.org/en/13/orm/tutorial.html#common-filter-operators). Preference?

Reply:
I think either `.is_(None)` or put the comment that disables the rule right on the relevant line. It was confusing to see the comment at the top of the function.

Reply:
I'll go with `.is_(None)`. Trying to put it on the same line means you end up fighting with `black` since it wants to wrap the newly-long line and then the comment no longer applies to the right line.